scripts/testnet/sovereignBridge/config/esdt-safe.snippets.sh

```diff
@@ -2,167 +2,225 @@ ESDT_SAFE_ADDRESS=$(mxpy data load --partition=${CHAIN_ID} --key=address-esdt-sa
         exit 1
     fi
 
     local ADDRESS=$1
     local URL=$2
     local CHAIN=$3
+    local OUTFILE=$4
 
-    mxpy --verbose contract call ${ADDRESS} \
+    mxpy contract call ${ADDRESS} \
         --pem=${WALLET} \
         --proxy=${URL} \
         --chain=${CHAIN} \
         --gas-limit=10000000 \
         --function="unpause" \
+        --outfile=${OUTFILE} \
         --recall-nonce \
         --wait-result \
         --send || return
+
+    printTxStatus ${OUTFILE}
 }
 
 setFeeMarketAddress() {
-    setFeeMarketAddressCall ${ESDT_SAFE_ADDRESS} ${FEE_MARKET_ADDRESS} ${PROXY} ${CHAIN_ID}
+    echo "Setting Fee Market address in ESDT Safe contract on main chain..."
+
+    local OUTFILE="${OUTFILE_PATH}/set-feemarket-address.interaction.json"
+    setFeeMarketAddressCall ${ESDT_SAFE_ADDRESS} ${FEE_MARKET_ADDRESS} ${PROXY} ${CHAIN_ID} ${OUTFILE}
 }
 setFeeMarketAddressSovereign() {
-    setFeeMarketAddressCall ${ESDT_SAFE_ADDRESS_SOVEREIGN} ${FEE_MARKET_ADDRESS_SOVEREIGN} ${PROXY_SOVEREIGN} ${CHAIN_ID_SOVEREIGN}
+    echo "Setting Fee Market address in ESDT Safe contract on sovereign chain..."
+
+    local OUTFILE="${OUTFILE_PATH}/set-feemarket-address-sovereign.interaction.json"
+    setFeeMarketAddressCall ${ESDT_SAFE_ADDRESS_SOVEREIGN} ${FEE_MARKET_ADDRESS_SOVEREIGN} ${PROXY_SOVEREIGN} ${CHAIN_ID_SOVEREIGN} ${OUTFILE}
 }
 setFeeMarketAddressCall() {
-    if [ $# -lt 4 ]; then
-            echo "Usage: $0 <arg1> <arg2> <arg3> <arg4>"
-            exit 1
-        fi
+    if [ $# -lt 5 ]; then
+        echo "Usage: $0 <arg1> <arg2> <arg3> <arg4> <arg5>"
+        exit 1
+    fi
 
     local ESDT_SAFE_CONTRACT_ADDRESS=$1
     local FEE_MARKET_CONTRACT_ADDRESS=$2
     local URL=$3
     local CHAIN=$4
+    local OUTFILE=$5
 
-    mxpy --verbose contract call ${ESDT_SAFE_CONTRACT_ADDRESS} \
+    mxpy contract call ${ESDT_SAFE_CONTRACT_ADDRESS} \
         --pem=${WALLET} \
         --proxy=${URL} \
         --chain=${CHAIN} \
         --gas-limit=10000000 \
         --function="setFeeMarketAddress" \
         --arguments ${FEE_MARKET_CONTRACT_ADDRESS} \
+        --outfile=${OUTFILE} \
         --recall-nonce \
         --wait-result \
         --send || return
+
+    printTxStatus ${OUTFILE}
 }
 
-setMultisigAddress() {
-    checkVariables ESDT_SAFE_ADDRESS MULTISIG_VERIFIER_ADDRESS || return
+setHeaderVerifierAddressInEsdtSafe() {
+    echo "Setting Header Verifier address in ESDT Safe contract on main chain..."
+    checkVariables ESDT_SAFE_ADDRESS HEADER_VERIFIER_ADDRESS || return
 
-    mxpy --verbose contract call ${ESDT_SAFE_ADDRESS} \
+    local OUTFILE="${OUTFILE_PATH}/set-headerverifier-address.interaction.json"
+    mxpy contract call ${ESDT_SAFE_ADDRESS} \
         --pem=${WALLET} \
         --proxy=${PROXY} \
         --chain=${CHAIN_ID} \
         --gas-limit=10000000 \
-        --function="setMultisigAddress" \
-        --arguments ${MULTISIG_VERIFIER_ADDRESS} \
+        --function="setHeaderVerifierAddress" \
```

Comment:
Why header verifier?
Since when are we not verifying multisigs anymore and only headers?

Reply:
Andrei renamed the contract with request from Robert.